README.md

```diff
@@ -48,8 +51,12 @@ A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marke
 
 This extension provides full Puppet Language support for [Visual Studio Code](https://code.visualstudio.com/).
 
+- Loading indicator
```

Comment:
Does this belong here? Seems reasonable to leave it out of the readme (but include in the changelog!) - not really a Puppet-centric feature, I think?

Reply:
I dunno. I think it's something nice to callout as it's helpful, but didn't know where to put it

Reply:
I think it can probably be excluded from here - the usage section below seems to handle it fine.

Reply:
```suggestion
```